packages/flutter/lib/src/gestures/force_press.dart

```diff
@@ -203,10 +206,17 @@ class ForcePressGestureRecognizer extends OneSequenceGestureRecognizer {
 
   @override
   void addPointer(PointerEvent event) {
-    startTrackingPointer(event.pointer);
-    if (_state == _ForceState.ready) {
-      _state = _ForceState.possible;
-      _lastPosition = event.position;
+    assert(event.pressureMax >= 1.0);
+    // If the device has a maximum pressure of less than or equal to 1, (ie. on
```

Comment:
Describe root principles. It's not because it's a non-iOS device that we're not calling callbacks. It's because there is no useful pressure data since every event is a max pressure event on devices with no pressure sensing hardware.